Formula/spatialite-gui.rb

```diff
@@ -24,20 +23,29 @@ class SpatialiteGui < Formula
   depends_on "pkg-config" => :build
   depends_on "freexl"
   depends_on "geos"
-  depends_on "libgaiagraphics"
+  depends_on "libpq"
+  depends_on "librasterlite2"
+  depends_on "librttopo"
   depends_on "libspatialite"
+  depends_on "libtiff"
+  depends_on "libxlsxwriter"
+  depends_on "lz4"
+  depends_on "minizip"
+  depends_on "openjpeg"
   depends_on "proj@7"
   depends_on "sqlite"
+  depends_on "virtualpg"
+  depends_on "webp"
   depends_on "wxwidgets@3.0"
```

Comment:
Since we're updating this we should check whether it supports the newer `wxwidgets`.

Reply:
Unfortunately I tried this and it doesn't work.  This is even mentioned in a comment in the AUR page for `spatialite-gui`: https://aur.archlinux.org/packages/spatialite-gui.